clang/include/clang/Driver/Options.td

```diff
@@ -2533,6 +2533,31 @@ def fno_sanitize_merge_handlers : Flag<["-"], "fno-sanitize-merge">, Group<f_cla
                         Alias<fno_sanitize_merge_handlers_EQ>, AliasArgs<["all"]>,
                         Visibility<[ClangOption, CLOption]>,
                         HelpText<"Do not allow compiler to merge handlers for any sanitizers">;
+def fsanitize_add_pseudo_functions_EQ
+    : CommaJoined<["-"], "fsanitize-add-pseudo-functions=">,
```

Comment:
Maybe `-fsanitize-add-pseudo-functions` -> `-fsanitize-pseudo-function` ?

Reply:
`-fsanitize-pseudo-function` is slightly ambiguous: it sounds like it is an option to sanitize any existing pseudo-functions, rather than sanitizers adding pseudo-functions

Reply:
convention is -fsanitize-<options>=<what we sanitize>, so I think it's not a problem.

But unless someone step in, either name is fine to me, your call.

Reply:
I agree with vitaly.

-fsanitize-trap doesn't sanitize traps either